geometry_msgs/Transform current_se3
geometry_msgs/Transform[] target_se3
float64 duration
int16 traj_type
float64[] kp
float64[] kv
std_msgs/Bool iswholebody
